pyvcloud/vcd/org.py

```diff
@@ -77,6 +77,32 @@ def create_catalog(self, name, description):
             EntityType.ADMIN_CATALOG.value,
             catalog)
 
+    def create_role(self, role_name, description, rights):
+        """
+        Creates a role in the organization
+        :param role_name: name of the role to be created
+        :param description: description of the role
+        :param rights: names of zero or more rights to be associated with the role
+        :return: RoleType just created
+        """  # NOQA
+        org_admin_resource = self.client.get_resource(self.href_admin)
+        role = E.Role(
+            E.Description(description),
+            E.RightReferences(),
+            name=role_name
+        )
+        for right in rights:
```

Comment:
Before iterating, I think we should make sure that rights is a list. Also a Null check should be made.

Reply:
I changed:  for right in rights -> for right in tuple(rights). This doesn't require None check.